lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -290,12 +290,6 @@ class EngineParagraph implements ui.Paragraph {
     final int end = WordBreaker.nextBreakIndex(_plainText, offset);
     return <int>[start, end];
   }
-
-  @override
-  List<ui.LineMetrics> computeLineMetrics() {
```

Comment:
Return empty List here?

Reply:
Done. Should be good now. Once and for all!